src/ray/common/ray_config_def.h

```diff
@@ -469,4 +469,7 @@ RAY_CONFIG(uint64_t, resource_broadcast_batch_size, 512);
 RAY_CONFIG(bool, worker_resource_limits_enabled, false)
 
 /// ServerCall instance number of each RPC service handler
-RAY_CONFIG(int64_t, gcs_max_active_rpcs_per_handler, 100)
\ No newline at end of file
+RAY_CONFIG(int64_t, gcs_max_active_rpcs_per_handler, 100)
+
+/// Whether to use log reporter in event framework
+RAY_CONFIG(bool, event_log_reporter_enabled, true)
```

Comment:
Can we turn this off by default until we solve the thread safety issue? I am a little worried the logging.cc change can cause memory corruption in some scenarios (since logging module is shared by all threads)

Reply:
Make sense.